packages/mdc-list/mdc-list.scss

```diff
@@ -191,7 +170,13 @@ a.mdc-list-item {
 }
 
 @at-root {
-  @include mdc-list-divider-color($mdc-list-divider-color-light);
+  $divider-color: if(
+    mdc-theme-contrast-tone($mdc-theme-background) == "dark",
```

Comment:
My mistake - this should actually be `mdc-theme-tone($mdc-theme-background) == "dark"`

The `$mdc-list-divider-color-[light|dark]` variables are named confusingly, so I got them mixed up.

If you look at the "Full-Width Dividers" and "Inset Dividers" examples, the dividers should be visible; currently they're not (because the colors were flipped).

Reply:
yup just noticed...looks better